tools/statistics/gen_yaml_for_return_value_checks.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+TEMP=`getopt -o t: --long threshold -n gen_yaml_for_return_value_checks.sh -- "$@"`
```

Comment:
how would this work with running multiple clang instances in parallel? Should we collect the ouput of clang  instances in to an intermediary file and then, as a post processing step echo this file into this shell script which would calculate the percentages based on that?